### PR TITLE
Mapped children

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
* big improvements to speed and rendering of flow run and task run schematics with mapped tasks
* flow run schematics no longer query for mapped task runs with indexes > -1
* task run dependency schematics are smarter, dynamically generating `mapped_children` queries for all tasks in the immediate tree
*  adds a new tab to the task run page that allows searching through a table of all children/siblings for a given mapped task
* converts expected mapped states to locale strings wherever possible
* fix bug where the task run activity tile was randomly resorting itself based on the `updated` column (now uses the `state_timestamp` column for sorting)
* add task run names to the schematics tiles where possible
* cleans up the flow run and task run queries (there were unused joins 😬 )

### CODE REVIEW

File: src/pages/TaskRun/Dependencies-Tile.vue [hunk 1]
```diff
@@ -47,50 +84,104 @@ export default {
...
+          task.task.downstream_edges = task.task.downstream_edges.filter(
+            edge => this.taskRun.task.id == edge.downstream_task.id
+          )
+          return task
+        })
+
+        dependencies.forEach(d => {
+          d.flow_run_name = this.taskRun.flow_run.name
+
+          d.task.name =
```
Comment (zhen0) [inline src/pages/TaskRun/Dependencies-Tile.vue:177]: Is it possible that we're adding the map index twice? (Here and also in the schematic flow?). 
Screen Shot 2020-10-22 at 11 03 43 AM

File: src/components/Schematics/Schematic-Node.vue [hunk 2]
```diff
@@ -4,12 +4,12 @@ import DurationSpan from '@/components/DurationSpan'
 import StackedLineChart from '@/components/Visualizations/StackedLineChart'
 import { STATE_COLORS } from '@/utils/states'
 
-const defaultSegments = () => [
-  { label: 'Pending', value: 0 },
-  { label: 'Running', value: 0 },
-  { label: 'Success', value: 0 },
-  { label: 'Failed', value: 0 }
-]
+// const defaultSegments = () => [
```
Comment (zhen0) [inline src/components/Schematics/Schematic-Node.vue:7]: Should we take out the comments?

**Review by zhen0 — COMMENTED** [review]
A couple of small comments/checks

File: src/pages/TaskRun/Dependencies-Tile.vue (hunk 1 above)
Comment (znicholasbrown) [reply src/pages/TaskRun/Dependencies-Tile.vue:179]: Yup, nice catch!

**Comment by znicholasbrown** [comment]
Comments addressed @zhen0 !

**Review by zhen0 — APPROVED** [review]
Thanks @znicholasbrown

**Review by cicdw — COMMENTED** [review]
Two comments:

- the Gantt chart remains a white / blank screen on the 50,000 mapped flow; not sure if that's known or expected
- it would be nice to display the state statistics more visibly in this dialogue box instead of only displaying them when hovering: 
image

**Comment by znicholasbrown** [comment]
Updates have been made as discussed offline, this is RTG @cicdw

**Review by cicdw — APPROVED** [review]

**Pull request merged**